### PR TITLE
Feature/browserify

### PR DESCRIPTION
**Pull request by TheAddictedOne**
Basic usage of Browerify.
Do not hesitate to ask questions if comments are not clear enough.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,16 @@
   "description": "Coach app for Pokemon Go who advise you which Pokemon to take against another",
   "main": "front/index.html",
   "devDependencies": {
+    "babel-core": "^6.14.0",
```
Comment (Shyrion) [inline package.json]: Please remove this ^ by using --save-exact

File: front/scripts/TYPE_TO_CSS_CLASS.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const TYPE_TO_CSS_CLASS = [
```
Comment (Shyrion) [inline front/scripts/TYPE_TO_CSS_CLASS.js]: No export keyword?

File: front/scripts/Polyfills.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+/**
+ * Set Polyfills as a static 'class'.
+ * It's just a static Object, like Math.
+ * It's a pattern which don't need to be extract from main, if it's usage is only to fix
+ * Object.assign existance once
+ */
+export default class Polyfills {
+  /**
+   * Your olf method, wrap in a static method
```
Comment (Shyrion) [inline front/scripts/Polyfills.js]: old*

**Comment by Shyrion** [comment]
Awesome PR :). I have something similar using `watchify` currently, but would gladly merge this PR once all comments are clarified 👍

File: front/scripts/TYPE_TO_CSS_CLASS.js (hunk 2 above)
Comment (TheAddictedOne) [inline front/scripts/TYPE_TO_CSS_CLASS.js]: Right. Too fast and furious commit :)

File: package.json (hunk 1 above)
Comment (TheAddictedOne) [inline package.json]: Right. Bad habit with `--save-dev`

File: front/scripts/Polyfills.js (hunk 3 above)
Comment (TheAddictedOne) [inline front/scripts/Polyfills.js]: +1

**Pull request merged**